app/src/main/java/com/owncloud/android/authentication/AuthenticatorActivity.java

```diff
@@ -1379,6 +1391,8 @@ public void onRequestPermissionsResult(int requestCode,
      * server.
      */
     private void showServerStatus() {
+        if (accountSetupBinding == null) return;
```

Comment:
What is this about? This should never happen.

Reply:
There are some cases in which this method is directly called without checking binding null.

Reply:
Okay, can you upstream this?

Reply:
Upstreamed: https://github.com/nextcloud/android/pull/11566